gcsmigrate/k8s/base/final_migrate_job.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: lemursync-final
+spec:
+  schedule: "5,15,25,35,45,55 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: rhino-sa
+          containers:
+            - name: lemursync-final
+              image: gcr.io/plos-mgmt/eng-462
+              args:
+                - /bin/sh
+                - -c
+                - "python enqueue.py final_migrate_rhino && python enqueue.py final_migrate_lemur"
+              volumeMounts:
+                - mountPath: /lemursync
+                  name: lemursync-data
+              envFrom:
+                - configMapRef:
+                    name: lemursync-configmap
+                - secretRef:
+                    name: lemursync
```

Comment:
a little pedantic but I usually call this something like 'lemursync-secrets' so you know it's a secret when referring to it later.